app/assets/stylesheets/cart.css.scss

```diff
@@ -88,3 +88,23 @@
   a:visited {color: #008000; text-decoration: underline; }
 }
 
+input[type="submit"].cart-item-quantity-button {
+  border : solid 0px #ffffff;
+  border-radius : .95em;
```

Comment:
`.95` should be written with a leading zero as `0.95`<br>Property name should be immediately followed by a colon